extensions/MyDashboard/web/js/flags.js

```diff
@@ -43,7 +43,7 @@ $(function () {
                 Y.one(`#${type}_loading`).addClass('bz_default_hidden');
                 Y.one(`#${type}_count_refresh`).removeClass('bz_default_hidden');
 
-                alert(`Failed to load requests:\n\n${message}`);
+                dataTable[type].showMessage(`Failed to load requests.`);
```

Comment:
Should we log the actual error message to console?

Reply:
I already have [`console.log()`](https://github.com/mozilla-bteam/bmo/blob/master/js/util.js#L642) 🙂